notebooks/Intertidal_CLI.ipynb

```diff
@@ -170,31 +299,68 @@
     "\n",
     "# Single timesteps (2019 - 2021)\n",
     "claire_ids =['107', '108', '109', '137', '138', '160', '161', '175', '176', '272', '273', '294', '295', '296', '318', '319', '297', '337', '338', '463', '464', '483', '477', '478', '494', '506', '507', '519', '638', '654', '655']#, '776']\n",
```

Comment:
```suggestion
    "claire_ids =['107', '108', '109', '137', '138', '160', '161', '175', '176', '272', '273', '294', '295', '296', '318', '319', '297', '337', '338', '463', '464', '483', '477', '478', '494', '506', '507', '519', '638', '654', '655', '776']\n",
```

Reply:
My first sample data run failed after tile 776 so I updated my tileset to avoid re-running it. This edit returns it to the full sample data run tileset.